### PR TITLE
HParams: Implement selection checkbox double click

### PR DESCRIPTION
**Pull request by JamesHollyer**
## Motivation for features / changes
We will soon be replacing the RunsTableComponent with the RunsDataTable. In an effort to reach feature parity with the current runs table this PR implements the dblClick functionality of the selected check boxes. When the checkboxes are double clicked it has the effect of selecting this and only this run.

## Technical description of changes
I implemented the dblclick logic differently than the RunsTableComponent. I utilized the MouseEvent details property to decide if a click was a click or a dblclick. This stops the click from firing both the onSelectionToggle and the onSelectionDblClick events for a single click.

### CODE REVIEW

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts [hunk 1]
```diff
@@ -707,9 +709,10 @@ export class RunsTableContainer implements OnInit, OnDestroy {
     // (see https://www.quirksmode.org/dom/events/click.html). We presume, then,
     // that we can rely on the 'change' event being fired before the 'dblclick'
     // event.
+    console.log('onRunSelectionDblClick', runId);
```
Comment (bmd3k) [inline tensorboard/webapp/runs/views/runs_table/runs_table_container.ts:712]: Remove console.log() call.

Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_table_container.ts]: Done.

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.ts [hunk 2]
```diff
@@ -88,6 +89,22 @@ export class RunsDataTable {
     );
   }
 
+  selectionClick(event: MouseEvent, runId: string) {
+    // Prevent checkbox from switching checked state on its own.
+    event.preventDefault();
```
Comment (bmd3k) [inline tensorboard/webapp/runs/views/runs_table/runs_data_table.ts:94]: If I recall correctly, I attempted this approach for the original implementation but couldn't get it to work properly. I would guess I was missing this call to preventDefault().

Nice work.

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts [hunk 3]
```diff
@@ -694,11 +695,12 @@ export class RunsTableContainer implements OnInit, OnDestroy {
     );
   }
 
-  onRunSelectionDblClick(item: RunTableItem) {
-    // Note that a user's double click will trigger both 'change' and 'dblclick'
-    // events so onRunSelectionToggle() will also be called and we will fire
-    // two somewhat conflicting actions: runSelectionToggled and
-    // singleRunSelected. This is ok as long as singleRunSelected is fired last.
+  onRunSelectionDblClick(runId: string) {
+    // Note that a user's double click in the Legacy RunsTableComponent will
```
Comment (bmd3k) [inline tensorboard/webapp/runs/views/runs_table/runs_table_container.ts:699]: Not sure if it's worth doing but it would be sort of nice to fix the legacy runs_table_component to use a similar implementation to the runs_data_table,  just so you could delete this entire comment. (If you decide to do it, it could be left to a different PR).

**Review by bmd3k — APPROVED** [review]

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.ts (hunk 2 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_data_table.ts:94]: Thanks

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts (hunk 3 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_table_container.ts:699]: I thought that would be a good follow up as well.
